### PR TITLE
[Vue3-XState] README documentation

### LINKED ISSUE #514: [Vue3 + XState] Update readme
Acceptance criteria: 

- [ ] Default readme is replaced with a specific one for this kit
- [ ] Required details on the tech stack, architectural details, and how to run the kit are included

### PR DESCRIPTION
**Pull request by lindakatcodes**
## Type of change

- [x] Documentation change
- [ ] Bug fix

## Summary of change

Updates the README to include relevant info about this kit. Also moves the component tests into their own folder for better consistency in how the project is structured (decided while writing the readme).

## Checklist

- [x] These changes follow the [contributing guidelines](https://github.com/thisdot/starter.dev/blob/main/CONTRIBUTING.md)
- [x] This starter kit has been approved by the maintainers
- [x] Changes for this new starter kit are being pushed to an integration branch instead of main
- [x] This fix resolves #514

### CODE REVIEW

File: starters/vue3-xstate-css/README.md [hunk 1]
```diff
@@ -1,68 +1,204 @@
...
+```
 
-## Type Support for `.vue` Imports in TS
+<sub>(use any of `npm` / `yarn` / `pnpm`)</sub>
 
-TypeScript cannot handle type information for `.vue` imports by default, so we replace the `tsc` CLI with `vue-tsc` for type checking. In editors, we need [TypeScript Vue Plugin (Volar)](https://marketplace.visualstudio.com/items?itemName=Vue.vscode-typescript-vue-plugin) to make the TypeScript language service aware of `.vue` types.
+If you'd prefer manual installation, you can follow this process:
 
-If the standalone TypeScript plugin doesn't feel fast enough to you, Volar has also implemented a [Take Over Mode](https://github.com/johnsoncodehk/volar/discussions/471#discussioncomment-1361669) that is more performant. You can enable it by the following steps:
+> Note: The command below clones the entire repository, including other starter kits.
```
Comment (BigAB) [inline starters/vue3-xstate-css/README.md:15]: Personally, I feel offering this seconds way of cloning the entire repository is bit of a waste of the valuable above the fold space.  The people who would want to clone the entire repository, probably already know how to, and so, I would remove this and replace it with a link to a wiki entry for it

File: starters/vue3-xstate-css/README.md [hunk 2]
```diff
@@ -1,68 +1,204 @@
...
-## Customize configuration
+Copy and rename the "starters/vue3-xstate-css" directory to the name of your new project.
 
-See [Vite Configuration Reference](https://vitejs.dev/config/).
+## Tech Stack
 
-## Project Setup
+- [Vue 3](https://vuejs.org/)
+- [XState](https://xstate.js.org/)
+- CSS
```
Comment (BigAB) [inline starters/vue3-xstate-css/README.md:27]: IMHO we probably don't need to have CSS as the stack, the use of Vue implies it

File: starters/vue3-xstate-css/README.md [hunk 3]
```diff
@@ -1,68 +1,204 @@
...
+### Development
 
-This runs the end-to-end tests against the Vite development server.
-It is much faster than the production build.
+Once you've generated your new starter kit following the instructions above:
 
-But it's still recommended to test the production build with `test:e2e` before deploying (e.g. in CI environments):
+- `cd` into your project directory and run `npm install` to install the dependencies.
+- Run `npm run dev` to start the development server.
+- Click the <http://localhost:5173> link from within your terminal. This will open that link in a tab to view the application, and also launch the XState inspector for you in a second tab.
```
Comment (BigAB) [inline starters/vue3-xstate-css/README.md:56]: `<http://localhost:5173>` I didn't know you could make a link that way, neat

**Review by BigAB — COMMENTED** [review]
I made some comments, but they are all just opinions, so not requests for changes

File: starters/vue3-xstate-css/README.md (hunk 2 above)
Comment (lindakatcodes) [reply starters/vue3-xstate-css/README.md:27]: You're right that it's maybe not fully necessary, but it keeps the consistency of listing the UI framework, state mgmt, and styling selections in this section. Even though it's not a specific framework/library, I think pointing out that it uses pure CSS can still be helpful info.

**Review by william-mimura-thisdot — APPROVED** [review]
LGTM

**Pull request merged**